test/linalg3.jl

```diff
@@ -161,7 +161,10 @@ for elty in (Float32, Float64, Complex64, Complex128)
                                               0  -0.000000000000002   3.000000000000000])
 end
 
+OLD_STDERR = STDERR
+redirect_stderr()
 for elty in (Float64, Complex{Float64})
+    
```

Comment:
Yes, the second build (ARCH=x86_64, JULIA_CPU_CORES=1) timed out after 39:55. The last lines it printed are
```
c:\projects\julia\base\precompile.jl
 g++ -march=x86-64 -m64  -shared  -L/c/projects/julia/usr/lib/julia -L/c/projects/julia/usr/lib -L/c/projects/julia/usr/bin -o /c/projects/julia/usr/lib/julia/sys.dll /c/projects/julia/usr/lib/julia/sys.o -ljulia $([ WINNT = WINNT ] && echo '' -lssp)
make[1]: Leaving directory `/c/projects/julia'
make[1]: Entering directory `/c/projects/julia/test'
 gcc -march=x86-64 -m64  -O0 -ggdb3 -DJL_DEBUG_BUILD -fstack-protector-all -O3 ccalltest.c  -shared -o libccalltest.dll  -DCC="gcc -march=x86-64 -m64"
make[1]: Leaving directory `/c/projects/julia/test'
usr\bin\julia -e "versioninfo()"
```